recipes/conda-suggest-conda-forge/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "conda-suggest-conda-forge" %}
+{% set version = "2020.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/regro/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 942c655d31e6b9899c97f9ca2ed29ef3f879b6e9dc93635d24e446da02c871d6
+
+build:
+  number: 0
+
+test:
+  commands:
+    - test -f $PREFIX/share/conda-suggest/conda-forge.noarch.map  # [unix]
+    - test -f $PREFIX/share/conda-suggest/conda-forge.{{ build_platform }}.map  # [unix]
+    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.noarch.map exit 1  # [win]
+    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.{{ build_platform }}.map exit 1  # [win]
```

Comment:
```suggestion
    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.{{ target_platform }}.map exit 1  # [win]
```

Reply:
I can fix this up in the feedstock, but target_platform doesn't seem to be listed in the conda-build docs as the way to get the subdir name.  Where is this coming from? conda-build-config?

Reply:
It's from `conda-build`. `build_platform` is where this package is built in. `target_platform` is where the package ends up in.

Reply:
It seems to be undocumented, so probably not safe to use

Reply:
`build_platform` is wrong here though.

Reply:
Can you open an issue in `conda-build` to document `target_platform`?

Reply:
I agree, but as long as build and target are the same, it should be OK. I am a little overwhelmed right now trying to get through a rather large checklist.